src/amplitude-client.js

```diff
@@ -1258,8 +1258,8 @@ AmplitudeClient.prototype._limitEventsQueued = function _limitEventsQueued(queue
  * Note: the server response code and response body from the event upload are passed to the callback function.
  * @example amplitudeClient.logEvent('Clicked Homepage Button', {'finished_flow': false, 'clicks': 15});
  */
-AmplitudeClient.prototype.logEvent = function logEvent(eventType, eventProperties, opt_callback) {
-  if (this._shouldDeferCall()) {
+AmplitudeClient.prototype.logEvent = function logEvent(eventType, eventProperties, opt_callback, forceDiffer) {
```

Comment:
as I understand from the issue and from the conversation with Hao, this is `forceDefer` right? :)